### ISSUE TITLE
LambdaResNet Implementation?

### ISSUE THREAD

**Issue by McCrearyD** [lucidrains/lambda-networks#23]
I have been looking around and found [one implementation of LambdaResNets](https://github.com/leaderj1001/LambdaNetworks), although there seem to be some metric performance problems and I've found wall-clock performance problems (runs ~7x slower than normal resnets).

Do you plan on putting out a lambdaresnet model in this repository?

**Comment by lucidrains** [comment]
Hi! I have no plans as my research interests rests in attention rather than convolutions

Feel free to submit a PR though!

**Comment by d-li14** [comment]
@lucidrains Here is an implementation of Lambda ResNet with training logs and pre-trained models on ImageNet https://github.com/d-li14/lambda.pytorch. Would you please consider adding it to this repo?

**Comment by phongnhhn92** [comment]
> @lucidrains Here is an implementation of Lambda ResNet with training logs and pre-trained models on ImageNet https://github.com/d-li14/lambda.pytorch. Would you please consider adding it to this repo?

@d-li14  Did your implementation require a lot of memory to train?

**Comment by innat** [comment]
Here is another attempt to integrate lambda-net into a pre-trained model. Work in progress though, https://github.com/innat/LamdaDenseNet-121
